cartesi-rollups/api/backend/vouchers.md

```diff
@@ -13,12 +14,12 @@ For instance, in a DeFi application built on Cartesi, users may want to swap one
 
 Once generated, the voucher is transmitted to the base layer. This transmission signals the dApp's intent to execute the specified action on-chain.
```

Comment:
This is strange. The voucher is not "transmitted". The voucher exists since its creating, and can be executed on the base layer after the epoch closes.